clients-mapreduce/src/test/java/uk/gov/gchq/palisade/clients/mapreduce/InputFormatUtilsTest.java

```diff
@@ -46,41 +40,6 @@
 //        reqResponse.originalRequestId(new RequestId().id("test"));
```

Comment:
whats the status of this @BeforeClass and setup? if commented out can it be removed?

Reply:
They are part of the set-up for other commented-out tests